kit/scrolls/src/wrapScrollableComponent/wrapScrollableComponent.android.tsx

```diff
@@ -56,6 +81,34 @@ export function wrapScrollableComponent<Props extends ScrollViewProps>(
             return ref?.current;
         });
 
+        const automaticInsets =
+            automaticallyAdjustContentInsets || automaticallyAdjustKeyboardInsets;
+
+        const [internalContentInset, setInternalContentInset] = React.useState<Insets>(
```

Comment:
This might break iOS when contentInset is already applied (i.e. for disabled automaticInsets)

Reply:
The file is `wrapScrollableComponent.android.tsx`, so it won't be applied on iOS

Reply:
Oh, I found that it's andoid.tsx